build.gradle

```diff
@@ -87,37 +102,69 @@ pmd {
     ruleSets = []
 }
 
-findbugs {
+spotbugs {
     ignoreFailures = true
     effort = 'max'
     reportLevel = 'medium'
-    toolVersion = '3.0.1'
-    excludeFilterConfig = resources.text.fromArchiveEntry(configurations.codeMetrics, "findbugs/findbugs-exclude.xml")
+    toolVersion = '4.0.0'
+    excludeFilter = new File(rootDir, "config/metrics/findbugs/findbugs-exclude.xml")
```

Comment:
Is it safe to just assume the config is extracted to `config/metrics`? And should we also rename that folder inside to `spotbugs`?

Reply:
Not really safe. I made & published binaries from my local workspace without realizing this had been hooked up, it just didn't crash since I didn't trigger the analytics, heh. In the main Terasology engine the extract task is tied to some stuff to make sure it runs, we should do that here as well.